.travis.yml

```diff
@@ -111,6 +111,33 @@ jobs:
         - EXTRA_CXXFLAGS="-DDEBUG"
       script: echo "Not running any tests for a debug build."
 
+    # Ubuntu Linux with glibc using clang++-3.7, no-debug mode
+    - stage: Test different OS/CXX/Flags
```

Comment:
Have all the concerns about Travis being too slow been remedied in some way?

Reply:
@thk123 should answer this. I don't mind if the build with `-NDEBUG -DCPROVER_INVARIANT_DO_NOT_CHECK` is done by adding a new build or modifying an existing one. It is very clear, however, that such a build is needed.